### PR TITLE
[BAEL-5420] Added value object creation

### PR DESCRIPTION
**Pull request by mayank4all**

### CODE REVIEW

File: core-java-modules/core-java-lang-4/src/main/java/com/baeldung/employee/EmployeeBean.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package com.baeldung.employee;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+
+public class EmployeeBean implements Serializable {
```
Comment (andrzejsydor) [inline core-java-modules/core-java-lang-4/src/main/java/com/baeldung/employee/EmployeeBean.java:6]: just `Employee`? Why Bean?
```suggestion
public class Employee implements Serializable {
```

File: core-java-modules/core-java-lang-4/src/main/java/com/baeldung/employee/EmployeeVO.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package com.baeldung.employee;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class EmployeeVO {
+    public String firstName;
```
Comment (andrzejsydor) [inline core-java-modules/core-java-lang-4/src/main/java/com/baeldung/employee/EmployeeVO.java:7]: why `public`?
Should be `private` like `startDate`

File: core-java-modules/core-java-lang-4/src/main/java/com/baeldung/employee/EmployeePOJO.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
+package com.baeldung.employee;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+public class EmployeePOJO {
+
+    public String firstName;
```
Comment (andrzejsydor) [inline core-java-modules/core-java-lang-4/src/main/java/com/baeldung/employee/EmployeePOJO.java:8]:
```suggestion
    private String firstName;
```

**Review by andrzejsydor — COMMENTED** [review]
could be improve a bit

File: core-java-modules/core-java-lang-4/src/main/java/com/baeldung/employee/EmployeeBean.java (hunk 1 above)
Comment (mayank4all) [reply core-java-modules/core-java-lang-4/src/main/java/com/baeldung/employee/EmployeeBean.java:6]: > 

We are showing the conversion of POJO to a JavaBean. Using the name EmployeeBean gives more context to the readers.

**Pull request merged**